content/metadata.yaml

```diff
@@ -171,8 +171,11 @@ authors:
     initials: MDK
     orcid: 0000-0003-1258-5221
     email: mkessl11@jhmi.edu
+    funders:
+        - National Institutes of Health (R01DE027809-01A1S1)
```

Comment:
```suggestion
        - National Institutes of Health (R01DE027809)
```
This part of the grant number may be sufficient.  It's up to you.

Reply:
I was wondering the same thing - my PI thinks the full one is better for this grant.

Reply:
Ahh - the work was done only under the administrative supplement? That's my guess by adding the supplement ID.

Reply:
Makes sense.  Please disregard my comment.